_tuning-your-cluster/availability-and-recovery/workload-management/wlm-feature-overview.md

```diff
@@ -0,0 +1,203 @@
+---
+
+Introduced 2.18
+{: .label .label-purple }
+
+# Workload management
+
+Workload management allows users to group search traffic and isolate network resources, preventing the overuse of network resources by specific requests. It offers the following benefits:
+
+- Tenant-level admission control and reactive query management. When resource usage exceeds configured limits, it automatically identifies and cancels demanding queries, ensuring fair resource distribution.
+
+- Tenant-level isolation within the cluster for search workloads, operating at a node level.
+
+## Installing workload management
+
+To install workload management, use the following command: 
+
+```json
+./bin/opensearch-plugin install workload-management
+```
+{% include copy-curl.html %}
+
+## Permissions
+
+Only users with administator-level permissions can use workload management. 
+
+## Query groups
+
+A _query group_ is a logical group of tasks with defined resource limits. System administrators can dynamically manage query groups using the Workload management APIs. These query groups can be used to make search requests with resource limits. 
+
+### Operating modes
+
+The following operating modes determine the operating-level for the query group:
+
+- **Disabled mode**: Workload management is disabled.
+
+- **Enabled mode**: Workload management is enabled and will cause cancellations and rejection once the query group’s configured thresholds are reached.
+
+- **Monitor_only mode** (Default): Workload management will monitor tasks but it will not cancel/reject any queries.
+
+### Example request
+
+The following example adds a query group with the named `analytics`:
+
+```json
+PUT _wlm/query_group
+{
+  “name”: “analytics”,
+  “resiliency_mode”: “enforced”,
+  “resource_limits”: {
+    “cpu”: 0.4,
+    “memory”: 0.2
+  }
+}
+```
+{% include copy-curl.html %}
+
+When creating a query group, make sure that the sum of the resource limits for a single resource, such as `cpu` or `memory`, does not exceed `1`.
+
+### Example response
+
+OpenSearch responds with the set resource limits and the `_id` for the query group:
+
+```json
+{
+  "_id":"preXpc67RbKKeCyka72_Gw",
+  "name":"analytics",
+  "resiliency_mode":"enforced",
+  "resource_limits":{
+    "cpu":0.4,
+    "memory":0.2
+  },
+  "updated_at":1726270184642
+}
+```
+
+## Using `queryGroupID`
+
+You can associate a query request with a `queryGroupID` to manage and allocate resources within the limits defined by the query group. By utilizing this ID, requests are routed and tracked under the query group, ensuring resource quotas and task limits are maintained.
+
+The following example query uses the `queryGroupId` to ensure that the query stays under that query group's resource limits:
+
+```json
+{
+  "_id":"preXpc67RbKKeCyka72_Gw",
+  "name":"analytics",
+  "resiliency_mode":"enforced",
+  "resource_limits":{
+    "cpu":0.4,
+    "memory":0.2
+  },
+  "updated_at":1726270184642
+}
+```
+{% include copy-curl.html %}
```

Comment:
The correct way to use the queryGroupId to make search request is to assign `queryGroupId` header along with a search request. for example

```
curl -XGET "http://localhost:9200/*/_search" -H 'queryGroupId: preXpc67RbKKeCyka72_Gw'
```

Reply:
Copy and paste error here. Fixing now.